cmd/operator-sdk/bundle/validate.go

```diff
@@ -132,33 +180,86 @@ func (c bundleValidateCmd) run() (err error) {
 		fields["container-tool"] = c.imageBuilder
 	}
 	logger := log.WithFields(fields)
-	if viper.GetBool(flags.VerboseOpt) {
-		log.SetLevel(log.DebugLevel)
+	if c.isToOutPutJSON() {
+		// Log as JSON instead of the default ASCII formatter.
+		log.SetFormatter(&log.JSONFormatter{})
+	} else {
+		if viper.GetBool(flags.VerboseOpt) {
+			log.SetLevel(log.DebugLevel)
+		}
 	}
-
 	val := bundle.NewImageValidator(c.imageBuilder, logger)
-
 	// Pull image if a tag was passed.
 	if c.imageTag != "" {
-		logger.Info("Unpacked image layers")
+		msg := "Unpacked image layers"
+		if c.isToOutPutJSON() {
+			jsonOutput.addLog(msg, nil)
+		} else {
+			logger.Info(msg)
+		}
 		err = val.PullBundleImage(c.imageTag, dir)
 		if err != nil {
-			logger.Fatalf("Error to unpacking image: %v", err)
+			msg := "error to unpacking image"
+			if c.isToOutPutJSON() {
+				jsonOutput.addLog(msg, err)
+				jsonOutput.printFatalJSON()
+			}
+			logger.Fatalf(msg+": %v", err)
 		}
 	}
 
 	// Validate bundle format.
 	if err = val.ValidateBundleFormat(dir); err != nil {
-		logger.Fatalf("Bundle format validation failed: %v", err)
+		msg := "Bundle format validation failed"
+		if c.isToOutPutJSON() {
+			if valErr, isValErr := err.(bundle.ValidationError); isValErr {
+				for _, err := range valErr.Errors {
+					jsonOutput.addLog(msg, err)
+				}
+				jsonOutput.printFatalJSON()
+			}
+			jsonOutput.addLog(msg, err)
+			jsonOutput.printFatalJSON()
+		}
+		logger.Fatalf(msg+": %v", err)
 	}
 
 	// Validate bundle content.
 	manifestsDir := filepath.Join(dir, bundle.ManifestsDir)
 	if err = val.ValidateBundleContent(manifestsDir); err != nil {
-		logger.Fatalf("Bundle content validation failed: %v", err)
+		msg := "Bundle content validation failed"
+		if c.isToOutPutJSON() {
+			if valErr, isValErr := err.(bundle.ValidationError); isValErr {
+				for _, err := range valErr.Errors {
+					jsonOutput.addLog(msg, err)
+				}
+				jsonOutput.printFatalJSON()
+			}
+			jsonOutput.addLog(msg, err)
+			jsonOutput.printFatalJSON()
+		}
+		logger.Fatalf(msg+": %v", err)
 	}
 
-	logger.Info("All validation tests have completed successfully")
+	msg := "All validation tests have completed successfully"
+	if c.isToOutPutJSON() {
+		jsonOutput.addLog(msg, nil)
+		jsonOutput.printJSON()
+	} else {
+		logger.Info(msg)
+	}
+	return nil
+}
 
+// isToOutPutJSON returns true when it should output json
+func (c bundleValidateCmd) isToOutPutJSON() bool {
+	return c.outputFormat == "json-alpha1"
+}
+
+// isValidOutPutFormat returns true when the value info
+func (c bundleValidateCmd) validateOutPutFlag() error {
```

Comment:
This can go in the `validate` method, where other flags are validated.

Reply:
true .. good spot.